setup.py

```diff
@@ -170,12 +170,20 @@ def repair_wheel(self):
             self.copy_file(repaired_wheel, wheel_path)
             print(f"Copied repaired wheel to: {wheel_path}")
      
+def get_version_from_git():
```

Comment:
is running a subprocess calling the git command a good decision here ? 
There si no better way ?

Reply:
Not, sure... but calling `git` is an easy and stable solution, but maybe not directly in setup.py as `git` is apparently not installed on the build docker images. Torch is running `git` [too](https://github.com/pytorch/pytorch/blob/72dde6e84b2d8615c6297dd3a4c1b7f0297f7fe2/tools/generate_torch_version.py#L16) but they approach to version generation is more elaborate and probably runs before the build on the host machine. Although adding a version.py like torch, makes a lot of sense. We could put there the build configuration of whispercpp too, it will simplify debugging later. So let me see if I can run a script right after checkout.